src/main/java/handlers/CameraHandler.java

```diff
@@ -16,14 +15,26 @@
     /**
      * The list of the cameras.
      */
-    private List<Camera> cameraList;
+    private List<Camera> cameraList = new ArrayList<>();
+    private InformationHandler informationHandler;
     private CameraChestDetector cameraChestDetector = new CameraChestDetector();
+    private boolean active;
 
     /**
-     * Constructor for the CameraHandler class.
+     * Constructor for CameraHandler without information.
      */
     public CameraHandler() {
-        cameraList = new ArrayList<>();
+        informationHandler = new InformationHandler();
+        active = false;
+    }
+
+    /**
+     * Constructor for the CameraHandler class with information.
+     * @param information The informationHandler.
```

Comment:
Maybe use a different parameter name for the informationhandler instead of information? As these two things mean something different right?